robottelo/ui/container.py

```diff
@@ -71,7 +71,22 @@ def create(self, resource_name, name, command, parameter_list):
                 {'main_tab_name': 'Environment', 'name': 'TTY', 'value': True},
             ]
         """
-        self.assign_value(locators['container.resource_name'], resource_name)
+        # send_keys() can't send left parenthesis (see SeleniumHQ/selenium#674)
```

Comment:
Replacing ' (' with nothing will lead to no results in resources filter list. Not sending parenthesis part will make it impossible to use self.click() as we need full name there. Avoiding passing the data with parenthesis is not possible too, as we don't generate such data, we generate some random name for compute resource and ' (Docker)' part is automatically added.